packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -24,6 +36,146 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
  * @name Memberships
  */
 const Memberships = SparkPlugin.extend({
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    // Create a common envelope that we will wrap all events in
+    this.eventEnvelope = {
+      resource: SDK_EVENT_RESOURCE.MEMBERSHIPS,
+      // id -- webhook id concept does not correlate to SDK socket event
+      // name -- webhook name concept does not correlate to SDK socket event
+      // targetUrl -- targetUrl concept does not correlate to SDK socket event
+      // secret -- secret concept does not correlate to SDK socket event
+      ownedBy: SDK_EVENT_OWNER.CREATOR,
+      status: SDK_EVENT_STATUS.ACTIVE,
+      created: new Date().toISOString(),
```

Comment:
This will send the same `created` date for all events, no?

Reply:
Yes, this is the desired behavior as part of an effort to have the SDK Event payloads match as closely as possible to the payload of the existing webhooks.  

Today each webhook payload has an "envelope" which includes info about the webhook itself including when it was created.  In the webhook world this corresponds to the initial POST /webhooks call.  In the SDK world it corresponds to the memberships.listen() call.